migrations/migrations/20180925215318-user-migration.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.createTable('users', {
```

Comment:
Use same name of file in `models` because property `freezeTableName: true` maintain original name declared with `define`, check [docs](http://docs.sequelizejs.com/manual/tutorial/models-definition.html#configuration) for more info